sormas-backend/src/main/java/de/symeda/sormas/backend/common/BaseAdoService.java

```diff
@@ -444,18 +457,30 @@ public ADO getByUuid(String uuid, boolean fetchReferences) {
 			return null;
 		}
 
-		CriteriaBuilder cb = em.getCriteriaBuilder();
-		ParameterExpression<String> uuidParam = cb.parameter(String.class, AbstractDomainObject.UUID);
-		CriteriaQuery<ADO> cq = cb.createQuery(getElementClass());
-		Root<ADO> from = cq.from(getElementClass());
-		if (fetchReferences) {
-			fetchReferences(from);
-		}
+		final CriteriaBuilder cb = em.getCriteriaBuilder();
+		final ParameterExpression<String> uuidParam = cb.parameter(String.class, AbstractDomainObject.UUID);
+		final CriteriaQuery<Long> cq = cb.createQuery(Long.class);
+		final Root<ADO> from = cq.from(getElementClass());
+
+		cq.select(from.get(AbstractDomainObject.ID));
+
 		cq.where(cb.equal(from.get(AbstractDomainObject.UUID), uuidParam));
 
-		TypedQuery<ADO> q = em.createQuery(cq).setParameter(uuidParam, uuid);
+		final TypedQuery<Long> q = em.createQuery(cq).setParameter(uuidParam, uuid);
+		final Optional<Long> idOptional = q.getResultList().stream().findFirst();
```

Comment:
I think we should use `QueryHelper.getSingleResult` here, more than one result is not supported/expected.